scss/base/_colors.scss

```diff
@@ -5,6 +5,21 @@ $white: #ffffff;
 .bg-white { background-color: $white; }
 .bc-white { border-color: $white; }
 
```

Comment:
Should we be worried about this? (from the top of this file) 🤔 

> // generated from ./src/lib/_colors.json by gulp-json-sass

Reply:
I'll take a look when i can build this 😅

Reply:
it seems fine? 🤔